rskj-core/src/test/java/co/rsk/peg/BridgeTest.java

```diff
@@ -193,6 +201,111 @@ public void registerBtcCoinbaseTransaction_after_RSKIP143_activation_null_data()
         Assert.assertNull(result);
     }
 
+    @Test
+    public void activeAndRetiringFederationOnly_activeFederationIsNotFromFederateMember_retiringFederationIsNull_throwsVMException() throws Exception {
+        BridgeMethods.BridgeMethodExecutor executor = Bridge.activeAndRetiringFederationOnly(null, null);
+
+        ActivationConfig activations = spy(ActivationConfigsForTest.genesis());
+        doReturn(true).when(activations).isActive(eq(RSKIP143), anyLong());
+
+        Federation activeFederationMock = PowerMockito.mock(Federation.class);
+
+        BridgeSupport bridgeSupportMock = mock(BridgeSupport.class);
+        doReturn(activeFederationMock).when(bridgeSupportMock).getActiveFederation();
+        doReturn(null).when(bridgeSupportMock).getRetiringFederation();
+
+        Bridge bridge = getBridgeInstance(bridgeSupportMock, activations);
+
+        PowerMockito.mockStatic(BridgeUtils.class);
+        when(BridgeUtils.isFromFederateMember(any(), eq(activeFederationMock))).thenReturn(false);
```

Comment:
instead of mocking this method what you could mock the rskTx to return a random address when executing `getSender`.

You are using `getBridgeInstance` which internally mocks the rskTx. You could create an overload of said method that allows you to provide a custom rskTx that you would be mocking outside.

Reply:
Done!